### DISCUSSION TITLE
22.0.0

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1474] [General]
## Releases Notes for 22.0.0
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.0/draw.io-22.0.0-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.0/draw.io-22.0.0-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.0/draw.io-universal-22.0.0.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.0/drawio-amd64-22.0.0.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.0/drawio-x86_64-22.0.0.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.0/drawio-x86_64-22.0.0.rpm)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- The major version change is due to a variable change in the core draw.io, that doesn't effect desktop.
- Updates to [draw.io core 22.0.0](https://github.com/jgraph/drawio/blob/v22.0.0/ChangeLog). All changes from 22.0.0 are added in this build.
- Fix for #1470 

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v22.0.0'>22.0.0</a>.</em>

**Comment by LinhUMD** [comment]
Is anyone else having problems with the MSI installer with the ALLUSERS flag? Looks like it keeps pushing to the users appdata folder still.
